src/applications/search/containers/SearchApp.jsx

```diff
@@ -155,6 +158,18 @@ class SearchApp extends React.Component {
       'search-results-top-recommendation': bestBet,
       'search-typeahead-enabled': this.props.searchTypeaheadEnabled,
     });
+
+    const encodedUrl = encodeURIComponent(url);
+    const userAgent = encodeURIComponent(navigator.userAgent);
+    const searchClickTrackingEndpoint = `/search_click_tracking`;
+    const apiRequestOptions = {
+      method: 'POST',
+    };
+
+    apiRequest(
+      `${searchClickTrackingEndpoint}?position=${searchResultPosition}&query=${query}&url=${encodedUrl}&user_agent=${userAgent}`,
```

Comment:
Does `query` need to be URL-encoded as well? It might already be encoded when pulled from the router's location (?)

Reply:
this should be pre-encoded from the search!